core/src/test/java/io/questdb/griffin/AlterTableAttachPartitionTest.java

```diff
@@ -746,6 +743,7 @@ public void testAttachPartitionsWithIndexedSymbolsValueMatch() throws Exception
         });
     }
 
+    @Ignore
```

Comment:
@marregui could you help me with updating this test to make it successfully passing? I wonder how I should update src and dst tables to get a column mismatch error of any kind.

Reply:
yes

Reply:
the test is correct, it passes.

The idea of the test is that we create two tables of different name, but same id. Other than the name the tables are identical, so it means we can use the metadata of one from the other.

Reply:
The expected exception is not thrown here, so this test fails on my machine. Am I missing something?

Reply:
I've reenabled the test in 91d338aeffc7efe59bd712743d329295b498f8d9 to demonstrate the failure.

Reply:
BTW you should be looking at `testAttachPartitionsWithSymbolsValueDoesNotMatch`. Based on your test scenario description, I assume that you were looking at another test.

Reply:
Nevermind, I've updated the test in 27e53ab2f2f5cce3b01884b0a8babd407605b5ea